### PR TITLE
Release note for Circle CI

### PR DESCRIPTION
**Pull request by nmohantyctr**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-service/2022-12-22-Circle-CI.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+Now you can report CircleCI job analytics to your SumoLogic dashboard.
+
+New job-collector command:
+This command sends environment, team, and service information at the job level. Add this command to your job with the environment, team, or service custom-data values as parameters. This command will run with the rest of the job's commands for sending the job log.
+
+:::
+Note: Not including this command at the job level will send empty custom-data values to Sumo.
+:::
```
Comment (kimsauce) [inline blog-service/2022-12-22-Circle-CI.md:21]: Broken note formatting

File: blog-service/2022-12-22-Circle-CI.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: Circle CI
+tags: [apps]
+hide_table_of_contents: true
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - sumo logic
+  - Circle CI
```
Comment (kimsauce) [inline blog-service/2022-12-22-Circle-CI.md:8]: CircleCI is one word

File: blog-service/2022-12-22-Circle-CI.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: Circle CI
```
Comment (kimsauce) [inline blog-service/2022-12-22-Circle-CI.md:2]: Please add more descriptive title

**Review by kimsauce — CHANGES_REQUESTED** [review]

**Review by kimsauce — CHANGES_REQUESTED** [review]
formatting/style edits

File: blog-service/2022-12-22-Circle-CI.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+title: CircleCI job analytics
```
Comment (kimsauce) [inline blog-service/2022-12-22-Circle-CI.md:2]:
```suggestion
title: CircleCI Job Analytics
```

File: blog-service/2022-12-22-Circle-CI.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - sumo logic
+  - CircleCI
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+Now you can report CircleCI job analytics to your SumoLogic dashboard.
```
Comment (kimsauce) [inline blog-service/2022-12-22-Circle-CI.md:14]:
```suggestion
Now you can report CircleCI job analytics to your Sumo Logic dashboard.
```

File: blog-service/2022-12-22-Circle-CI.md (hunk 1 above)
Comment (nmohantyctr) [reply blog-service/2022-12-22-Circle-CI.md]: Fixed

File: blog-service/2022-12-22-Circle-CI.md (hunk 3 above)
Comment (nmohantyctr) [reply blog-service/2022-12-22-Circle-CI.md]: Done

File: blog-service/2022-12-22-Circle-CI.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+New job-collector command:
+This command sends environment, team, and service information at the job level. Add this command to your job with the environment, team, or service custom-data values as parameters. This command will run with the rest of the job's commands for sending the job log.
```
Comment (kimsauce) [inline blog-service/2022-12-22-Circle-CI.md:17]:
```suggestion
You can now report CircleCI job analytics to your Sumo Logic dashboard using our new job-collector command, which sends environment, team, and service information at the job level. 

Add this command to your job with the environment, team, or service custom-data values as parameters. This command will run with the rest of the job's commands for sending the job log.
```

File: blog-service/2022-12-22-Circle-CI.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+:::note
+Not including this command at the job level will send empty custom-data values to Sumo.
+:::
+
+Also, the orb is now at sumologic/sumologic. The new orb repo is https://github.com/SumoLogic/sumologic-orb
```
Comment (kimsauce) [inline blog-service/2022-12-22-Circle-CI.md:23]:
```suggestion
To learn more, refer to the [Sumo Logic orb repo](https://github.com/SumoLogic/sumologic-orb#job-collector).
```

File: blog-service/2022-12-22-Circle-CI.md (hunk 2 above)
Comment (nmohantyctr) [reply blog-service/2022-12-22-Circle-CI.md]: Corrected.

**Review by kimsauce — APPROVED** [review]
looks good!

**Pull request merged**